src/views/MidiView.vue

```diff
@@ -178,9 +178,24 @@ onUnmounted(() => {
               />
             </span>
           </div>
-          <div class="control checkbox-container">
-            <input type="checkbox" id="midi-velocity" v-model="midi.velocityOn" />
-            <label for="midi-velocity">Use velocity</label>
+          <div class="control checkbox-group">
+            <span>
+              <input type="checkbox" id="midi-velocity" v-model="midi.velocityOn" />
+              <label for="midi-velocity"> Use velocity </label>
+            </span>
+            <span>
+              <input type="checkbox" id="multichannel" v-model="midi.multichannelToEquave" />
+              <label for="multichannel"> Multichannel-to-equave </label>
+            </span>
+          </div>
+          <label>Settings for multichannel-to-equave mode</label>
+          <div class="control dropdown-group">                    
```

Comment:
There's nothing *dropdown* about this group of elements. See `btn-dropdown-group` on what that means.